File: sdkwork-backend-sdk-go/types/plus_ai_prompt_vo.go
package types

// AI提示语Value Object
type PlusAiPromptVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Title string `json:"title"`
	Content string `json:"content"`
	Description string `json:"description"`
	CategoryId int `json:"categoryId"`
	Enabled bool `json:"enabled"`
	Sort int `json:"sort"`
	Parameters map[string]interface{} `json:"parameters"`
	Creator string `json:"creator"`
	Model string `json:"model"`
	Tags TagsContent `json:"tags"`
	UsageCount int `json:"usageCount"`
	AvgResponseTime int `json:"avgResponseTime"`
}

File: sdkwork-backend-sdk-csharp/Models/PlusAiPromptVO.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class PlusAiPromptVO
    {
        public string? CreatedAt { get; set; }
        public string? UpdatedAt { get; set; }
        public int? Id { get; set; }
        public string? Title { get; set; }
        public string? Content { get; set; }
        public string? Description { get; set; }
        public int? CategoryId { get; set; }
        public bool? Enabled { get; set; }
        public int? Sort { get; set; }
        public Dictionary<string, object>? Parameters { get; set; }
        public string? Creator { get; set; }
        public string? Model { get; set; }
        public TagsContent? Tags { get; set; }
        public int? UsageCount { get; set; }
        public int? AvgResponseTime { get; set; }
    }
}
